packages/@lwc/engine/src/framework/base-lightning-element.ts

```diff
@@ -304,47 +302,11 @@ function BaseLightningElementConstructor(this: LightningElement) {
 // HTML Element - The Good Parts
 BaseLightningElementConstructor.prototype = {
     constructor: BaseLightningElementConstructor,
-    dispatchEvent(event: Event): boolean {
+    dispatchEvent(): boolean {
```

Comment:
you should keep the first argument, just mark is with `_` to avoid the error of unused argument.